concept/tests/test_type.rs

```diff
@@ -157,6 +171,16 @@ fn entity_usage() {
         assert_eq!(root_entity.get_subtypes_transitive(&snapshot, &type_manager).unwrap().len(), 3);
         assert_eq!(person_type.get_subtypes(&snapshot, &type_manager).unwrap().len(), 2);
         assert_eq!(person_type.get_subtypes_transitive(&snapshot, &type_manager).unwrap().len(), 2);
+
+        // --- owns inheritance ---
+        let height_type = type_manager.get_attribute_type(&snapshot, &Label::new_static("height")).unwrap().unwrap();
+        match child_type.get_owns_attribute_transitive(&snapshot, &type_manager, height_type.clone()).unwrap() {
+            None => assert!(false, "child should inherit ownership of height"),
+            Some(child_owns_height) => {
+                assert_eq!(height_type, child_owns_height.attribute());
+                assert_eq!(ObjectType::Entity(person_type.clone()), child_owns_height.owner());
+            },
+        }
```

Comment:
Extra test. I could move it to a different test case and extend it to other inheritance, or let BDD do the work.